source/_integrations/iotawatt.markdown

```diff
@@ -12,8 +12,41 @@ ha_codeowners:
 ha_platforms:
   - sensor
 ---
-A sensor platform for the [IoTaWatt](https://www.iotawatt.com/) Open WiFi Electricity Monitor. It 
+A sensor platform for the [IoTaWatt](https://www.iotawatt.com/) Open WiFi Electricity Monitor. It
```

Comment:
```suggestion
Integration for the [IoTaWatt](https://www.iotawatt.com/) Open WiFi Electricity Monitor. It
```

While it's sensors only now, it is a full integration and not a sensor platform :man_shrugging: